simphony/scripts/cuba_generate.py

```diff
@@ -0,0 +1,99 @@
+import click
+import yaml
+
+# Cuba keywords that are excludes from DataContainers
+CUBA_DATA_CONTAINER_EXLCUDE = ['Id', 'Position']
```

Comment:
Couldn't be this an attribute in the yml? All other properties like shape, key,... are there,